spec/features/step_by_step_actions_spec.rb

```diff
@@ -67,12 +67,12 @@
   end
 
   def then_the_primary_action_should_be(action_text)
-    custom_error = "Couldn't find '#{action_text}' as a primary action in: \n #{action_html}"
-    expect(page).to have_css(".app-side__actions .gem-c-button:not(.gem-c-button--secondary)", text: action_text), custom_error
+    expect(page).to have_css(primary_action_selector, count: 1)
```

Comment:
It might be useful to have custom error text here, since the second expectation has this and it may not be clear what is required without reading the commit message

Reply:
The default error message is better, I think - it says something along the lines of:

> "Expected to find '.app-side__actions .gem-c-button:not(.gem-c-button--secondary)' 1 time. Found 3 times".

If I wrote a custom error text I'd be able to inject `action_html`, which is nice, but I'd also have to grab the actual vs expected count, which starts to get convoluted IMO.